### DISCUSSION TITLE
Starting Details for Location Management Team

### DISCUSSION THREAD

**Discussion by evnd04** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#8] [Informational]
Location Management Team responsibilities:

> Location reporting - That way, when the report is made, the coordinates of where it’s occurring are sent by as well
> Boundary setting and Coordinate Handling - Since it’s not ideal to just send coordinates as plain numbers, boundaries are set in place to recognize the coordinates within a range and mark it as the place’s name.
> Example: Instead of getting “18°12'34.6"N 67°08'24.5"W” in the report, it’s easier for the user to receive “Stefani Building” instead.
> Map Integration

provide your name, email, and username to be a part of this team, if interested in being a group leader add (team leader)

Team Leader: @Jjulianirizarry24 

Developers:
@misaelmercado1 
@JoseLQV 
@CaribbeanCool 

This team is currently: *FULL*

**Comment by misaelmercado1** [comment]
Misael J. Mercado
misaelmercado1
misael.mercado1@upr.edu

**Comment by JoseLQV** [comment]
Jose L. Quinones Velez
JoseLQV
jose.quinones27@upr.edu

**Comment by Jjulianirizarry24** [comment]
Jose J Irizarry Rivera
jjulianirizarry24
jose.irizarry24@upr.edu

**Comment by CaribbeanCool** [comment]
Alejandro A. Perez Pabon
CaaribbeanCool
alejandro.perez21@upr.edu

**Comment by YanehRuiz** [comment]
Hello everyone. This team consists of four people. Therefore I'd like to ask if any of you would like to step up as a team leader.
@misaelmercado1
@JoseLQV
@Jjulianirizarry24
@CaribbeanCool

**Reply by Jjulianirizarry24** [reply]
I'll do it :)

**Reply by YanehRuiz** [reply]
Perfect! I'll note you down ^^

**Comment by YanehRuiz** [comment]
This team is currently *FULL*!

**Comment by lex939** [comment]
**Message for: @misaelmercado1, @JoseLQV, @Jjulianirizarry24, and @CaribbeanCool**

Hi team!

Welcome to SafeRUM! I’m excited to work with you all on this project. 

From what I’ve read in the discussion so far, @Jjulianirizarry24 will be the team leader. I need two of the other members in the team to take on the responsibility of handling both documentation and software development for location management. These two members will split their time 50-50 between documentation and development, so each will do a bit less of each task to collectively meet the same total of tasks per milestone.

Please discuss among yourselves and let me know which people the team decides will fill these roles. Thanks so much!

**Comment by Jjulianirizarry24** [comment]
Misael Mercado and Alejandro Perez will be focusing on documentation.
